ntc_templates/templates/cisco_xr_show_bgp_vrf_all_neighbors_advertised-routes.textfsm

```diff
@@ -0,0 +1,20 @@
+Value NETWORK (\d+\.\d+\.\d+\.\d+)
+Value PREFIX_LENGTH (\d+)
+Value NEXT_HOP (\d+\.\d+\.\d+\.\d+)
+Value FROM ((\d+\.\d+\.\d+\.\d+)|\S+)
+Value AS_PATH (.*)
+Value Filldown ROUTE_DISTINGUISHER (\d+\:\d+)
+Value Filldown VRF (\S+)
+
+Start
+  ^\S{3}\s+\S{3}\s+\d{1,2}\s+\d+:\d+:\d+
+  ^Network\s+Next\s+Hop\s+From\s+AS\s+Path
```

Comment:
```suggestion
  ^Network\s+Next\s+Hop\s+From\s+AS\s+Path\s*$$
```

We should anchor this at the end of the line just in case the vendor were to add a column header to the right past AS Path.

I've added zero or more white space just in case there ever ends up being white space out to the right.